### ISSUE TITLE
oagtclsh used for some elegant utilities

### ISSUE THREAD

**Issue by cchall** [radiasoft/download#106]
A number of tools packaged with elegant do not function on Jupyter because they require oagtclsh (from https://www.aps.anl.gov/Accelerator-Operations-Physics/Software#PEM,%20Tcl/Tk I think)

computeGeneralizedGradients motived this issue, but there are others that are part of the list in the elegant manual in Section 8 (Specialized Tools for Use with elegant). Not all have this problem though. 

oagtclsh is normally just used in performing boilerplate setup that is either not strictly necessary or can be replaced by bash. I've manually removed it from scripts that I need in the past, but it is a tedious process - for me at least. Is there a better option for getting these running?

**Comment by robnagler** [comment]
So I was wrong. `oagtclsh` is simply a link to `/usr/bin/tclsh`. I have no idea why they do this, but it is referenced in every script so it has to be there.

What is important is where the `.tcl` libs are, and setting `$OAG_TOP_DIR`.

I will create a bash script called `oagtclsh` that will set `$OAG_TOP_DIR` properly. 

Sorry for the delay, but I had to wade through [Build-AOP-RPMs](https://ops.aps.anl.gov/downloads/Build-AOP-RPMs), [Build-AOP-RPMs.tcl](https://ops.aps.anl.gov/downloads/Build-AOP-RPMs.tcl), and [OAGTclTk.spec](https://ops.aps.anl.gov/downloads/OAGTclTk.spec) before I figured out what this line meant:

```txt
ln -sf %{_bindir}/tclsh %{buildroot}%{_bindir}/oagtclsh
```

It seems obvious now...

I need to figure out which `.tcl` files to install. There are lots.
